chain/client/src/info.rs

```diff
@@ -88,7 +88,7 @@ impl InfoHelper {
             * 1000.0;
         let avg_gas_used =
             ((self.gas_used as f64) / (self.started.elapsed().as_millis() as f64) * 1000.0) as u64;
-        info!(target: "info", "{} {} {} {} {} {}",
+        info!(target: "stats", "{} {} {} {} {} {}",
```

Comment:
so this is still log::info!

Reply:
Yes! But it can safely replaced with tracing without problem.
```
- use log::info;
+ use tracing::info;
```
I renamed target from `info` to `stats` because there were conflicts filtering with that name.